changelogs/fragments/821-vmware_guest_powerstate.yml

```diff
@@ -0,0 +1,3 @@
+minor_changes:
+  - vmware_guest_powerstate - Add an option that answers whether it was copied or moved the vm if the vm is blocked (https://github.com/ansible-collections/community.vmware/pull/821).
+  - vmware - add processing to answer if the answer question is occurred in starting the vm (https://github.com/ansible-collections/community.vmware/pull/821).
```

Comment:
I'm not sure about this changelog fragment. The change to `module_utils/vmware.py` is an internal change and doesn't affect any module _directly_.

Don't get me wrong, I'm not saying we shouldn't announce this change. But I'm quite sure I've seen a lot of (and done some myself) changes to `module_utils/vmware.py` without a changelog entry. So I'm ponderging: What's our policy here? Those who just _use_ this collection probably don't care about any internal changes like this. For them, `vmware_guest_powerstate - Add an option that answers whether it was copied or moved` is the important information. On the other hand, it _is_ a change and might be relevant for those who work on this collection.

@Akasurde @goneri Thoughts? Are there any guidelines whether to have changelog entries for module utils or not?

Reply:
As you said, I don't know the policy whether needing the vmware.py log, too.  
I have been seeing that sometimes there is the changelog of the vmware.py but sometimes there isn’t.  
But I think that the changelog is useful for the developers, so I think it's good if there is :-)

Reply:
As you say, the changelog is useful for the developers. So let's keep it. I just wanted to raise this point because I was curious if we have a policy on this or not.

I'll resolve this conversation for now. But, in future PRs, we might consider putting `module_utils/vmware` or `module_utils.vmware` in the changelog fragment. This would make it clearer that `vmware` is not a module, but something internal (a module util). Just an idea, not sure if it's a good one.